common/src/main/resources/reference.conf

```
@@ -2,7 +2,7 @@
 cluster-status-indicator {
 
   led-brightness = 10
-  led-count      = 8       // Number of LED pixels on LED strip
+  led-count      = 10       // Number of LED pixels on LED strip
```

(not a fan of manually aligning code, but if we do it let's do it right ;) )
```suggestion
  led-count      = 10      // Number of LED pixels on LED strip
```

---

does this belong in this PR anyway?

---

@raboof 
> does this belong in this PR anyway?

Good question; strictly speaking, it doesn't. I'll create a new PR targeting the current version on master that changes this.
Some context to clarify: when the new 10-LED strips became available, this change should have been applied immediately. Not doing so (so leaving this value at 8) will cause a problem as soon as one tries to control LED 9 or 10: it would throw an index out of range exception on an `Array`.